frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -65,6 +79,26 @@ const plugin: Plugin<ConsumedExtensions> = [
       template: yamlTemplates.getIn([models.VirtualMachineModel, 'default']),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: `/k8s/ns/:ns/${models.VmTemplateModel.plural}`,
```

Comment:
Is this needed? We have a resource list page and specialized model. I think generic router rule should work. Please correct me.

Reply:
This is what I understood from https://github.com/openshift/console/pull/1658#issuecomment-499574443 , I may have misunderstood the discussion ...

**Background**
AFAIU `VmTemplateModel` is of kind `Template` so defining it will redefining the existing model `Template`, so `ModelDefinition` skips it. Once I added the template we actually get a console warning:
```
attempt to redefine model template.openshift.io~v1~Template
```

Reply:
Yes, KubeVirt reuses the existing `Template` resource (`template.openshift.io~v1~Template`).

There's no need to redefine that model. What we need is a resource list page which applies a default filter to show VM templates only, e.g. via `Table.staticFilters` prop.

Reply:
> We have a resource list page and specialized model.

We don't have a specialized model for VM template, we reuse existing one.

Mapping between models vs. resource list/details pages is 1-1 so we need to use route-based page here.